init.c

```diff
@@ -981,6 +981,26 @@ static int fixup_options(struct thread_data *td)
 		}
 	}
 
+	for_each_td(td2) {
+		if (td->o.ss_check_interval != td2->o.ss_check_interval) {
+			log_err("fio: conflicting ss_check_interval: %llu and %llu, must be globally equal\n",
+					td->o.ss_check_interval, td2->o.ss_check_interval);
+			ret |= 1;
+		}
+	} end_for_each();
+	if (td->o.ss_dur && td->o.ss_check_interval / 1000L < 1000) {
+		log_err("fio: ss_check_interval %llu must be at least 1s,\n",
```

Comment:
drop the comma at the end of the error message or replace it with a period. might as well include the usec units in the error message as well since `fio: ss_check_interval 100000 must be at least 1s,` is confusing with no units